custom-script/utils.py

```diff
@@ -14,7 +18,38 @@
 MODEL_INPUTS_STR = ", ".join(MODEL_INPUTS)
 
 
-def parse_subtree(node, brackets=True):
+def prepare_data(eopatches: np.ndarray, train_eopatches: np.ndarray) -> Tuple[np.ndarray]:
+    FEATURES_SAMPLED = FeatureType.DATA, "FEATURES_SAMPLED"
+    IS_DATA_SAMPLED = FeatureType.MASK, "IS_DATA_SAMPLED"
+    LABELS_SAMPLED = FeatureType.MASK_TIMELESS, "water_label_SAMPLED"
```

Comment:
no reason to have them hard-coded in utility functions

Reply:
done